### DISCUSSION TITLE
Welcome to parsedmarc Discussions!

### DISCUSSION THREAD

**Discussion by seanthegeek** [domainaware/parsedmarc#544] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by superstes** [comment]
Greetings!

Thank you for developing this useful tool/service. 😊
It's crazy how much cloud provider want to get paid for functionality like this..

I've gave the codebase a look for https://github.com/domainaware/parsedmarc/issues/545 .
I personally think the huge functions make it really hard to understand and reverse-engineer what is going on.
[Will create a fork](https://github.com/O-X-L/dmarc-report-parser) and refactor it as I really enjoy refactoring (; Maybe it can spawn something useful for you upstream

**Reply by superstes** [reply]
Probable overlap with https://github.com/nhairs/parsedmarc-fork/issues/1

**Comment by makuartur** [comment]
Good day, could you please let me know if it is possible to store parsed data in S3 and build graphs based on this data? If yes, could you please advise on the best way to set this up?
